style.css

```diff
@@ -816,3 +926,15 @@ textarea {
     padding: 4px 15px;
   }
 }
+
+@media (min-width: 1000px) {
+  .container-popup {
+    margin: 2% 20%;
+    height: 88vh;
+  }
```

Comment:
- [ ] There is an issue between the buttons' position and the modal size, please 🙏🏽  fix it to be similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://github.com/alizaamir123/mobile-version/assets/58642949/4ade4fa7-de86-4acc-a632-42969dd345b8) | ![image](https://github.com/alizaamir123/mobile-version/assets/58642949/e0b02632-030b-42fd-82a5-5b0d0d45339c) |